app/views/media_objects/_share_resource.html.erb

```diff
@@ -29,6 +29,11 @@ Unless required by applicable law or agreed to in writing, software distributed
         <p class="muted"> <%= I18n.t('media_object.empty_share_section_permalink_notice') %> </p>
         <% end %>
       </div>
+      <div class="form-group">
+        <label for="link-object">IIIF Manifest</label>
+        <input type="text" class="col-md-12 form-control" id="link-object" value="<%= Rails.application.routes.url_helpers.manifest_media_object_url(@media_object.id) %>" 
```

Comment:
I noticed that this has the same ID as the field for the share link to the media object itself at ln 21-22. Would it be better to give them separate IDs?
```suggestion
        <label for="link-object-manifest">IIIF Manifest</label>
        <input type="text" class="col-md-12 form-control" id="link-object-manifest" value="<%= Rails.application.routes.url_helpers.manifest_media_object_url(@media_object.id) %>" 
```